src/Microsoft.Azure.WebJobs.Extensions.OpenApi/OpenApiTriggerFunctions.cs

```diff
@@ -32,7 +29,7 @@ public partial class OpenApiTriggerFunctionProvider
         /// <param name="log"><see cref="ILogger"/> instance.</param>
         /// <returns>OpenAPI document in a format of either JSON or YAML.</returns>
         [OpenApiIgnore]
-        public static async Task<IActionResult> RenderSwaggerDocument(HttpRequest req, string extension, ExecutionContext ctx, ILogger log)
+        public static async Task<IActionResult> RenderSwaggerDocument([OpenApiHttpTriggerContext] OpenApiHttpTriggerContext context, HttpRequest req, string extension, ExecutionContext ctx, ILogger log)

```

Comment:
Through the changes in OpenApiWebJobsStartup.cs, we can auto-inject the context which can be configured via Dependency Injection in consumers of the library :)